custom_components/kia_uvo/KiaUvoApiCA.py

```diff
@@ -124,9 +124,11 @@ def get_cached_vehicle_status(self, token: Token):
         vehicle_status["odometer"] = {}
         vehicle_status["odometer"]["unit"]= response["currentOdometerUnit"]
         vehicle_status["odometer"]["value"]= response["currentOdometer"]
-
-        vehicle_status["vehicleLocation"] = self.get_location(token)
-
+        try:
```

Comment:
let's postpone this, as we do not have any try/catch mechanism in overall integration, we can discuss what could be the best.
- having try catch in final functions: from development perspective, this is not good but easy to handle, one place to catch the errors
- having try catch in caller functions: from development perspective, this looks fine, but we might need to catch in multiple places depending on functions

Reply:
Done.  This one is important to me as I have to comment this line out to test.  The line works when not called too often though.